site/src/analytics-integrations.ts

```diff
@@ -20,6 +20,23 @@ import {
 
 const isMobile = matchMedia("(max-width: 600px)").matches;
 
+const IGNORED_DOMAINS = [
```

Comment:
Should we use a `Set`?

Reply:
With the amount of entries here, it does not really matter